test/helpers/candy-machine/asserts.ts

```diff
@@ -0,0 +1,23 @@
+import { Creator } from '@metaplex-foundation/mpl-token-metadata';
+import spok from 'spok';
+import { Test } from 'tape';
+import { CreatorsConfig } from '../../../src/modules/candy-machine/models/config';
+
+export function assertCreators(t: Test, creators: Creator[], config: CreatorsConfig) {
```

Comment:
I like the idea of having module-specific assertions on the tests but can we move that to `helpers/candy-machine.ts`. I don't want nested folders in `helpers`.

Reply:
These are asserts specific to candy machine, there might be more helpers specific to candy machine.
I'd like to avoid stuffing them all into one large file.